integration-test/helpers/harness.js

```diff
@@ -96,6 +95,11 @@ const setup = async (ops) => {
     return { browser, bgPage, requests, teardown, manifestVersion }
 }
 
+function getManifestVersion() {
```

Comment:
Why do we need this `getManifestVersion` function, if the `setup` function already returns the manifest version? If `setup` is returning the wrong manifest version, couldn't we fix that instead?

Reply:
setup is only run in the `beforeAll` step of the tests, but we check the manifest version already in the `describe` function of the tests (so it was always undefined). That means that setup happens after we already determined which tests to run.